lib/mix/tasks/start.ex

```diff
@@ -0,0 +1,6 @@
+defmodule Mix.Tasks.Start do
+  alias TicTacToe.CLI.Main, as: Main
```

Comment:
Nit, the `as` is not needed since the `alias` is just set to the last part of module

Reply:
OK! Added these aliases due to Credo violations but I will configure Credo to ignore

Reply:
Does the Credo check fail for you if you remove the `as`? I don't seem to run into any 👀

Reply:
Oh sorry I misread your comment. Got it passing now, thank you!!